.env.example

```diff
@@ -4,6 +4,9 @@ NODE_ENV="development"
 PORT=3000
 SESSION_SECRET="9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08"
 REDIS_URL="redis://redis:6379"
+SMTP_USERNAME="noreply.teamupapi@gmail.com"
+SMTP_PASSWORD="#TeamUp4Api!"
+FRONT_URL="https://localhost"
```

Comment:
we need to have CLIENT_URL and API_URL
CLIENT_URL="http://localhost"
API_URL="http://localhost/api"